src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
```

Comment:
Note: I assume that business logic is implemented correctly, as I have no knowledge of Credit Card validation.